src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -397,6 +397,17 @@ let setup_daemon logger =
   and node_status_url =
     flag "--node-status-url" ~aliases:[ "node-status-url" ] (optional string)
       ~doc:"URL of the node status collection service"
+  and node_error_url =
+    flag "--node-error-url" ~aliases:[ "node-error-url" ] (optional string)
+      ~doc:"URL of the node error collection service"
+  and contact_info =
+    flag "--contact-info" ~aliases:[ "contact-info" ] (optional string)
+      ~doc:
+        "contact info used in node error report service (it could be either \
+         email address or discord username)"
+    |> Command.Param.map ~f:(fun opt ->
+           Option.value_map opt ~default:None ~f:(fun s ->
+               if String.length s <= 200 then Some s else None))
```

Comment:
Report an error here, rather than silently failing. Mention the maximum length in the doc-string.

Reply:
`Mina_user_error.raisef` is probably the best 'type' of error to raise, since we handle it specially to do nice printing.